main/src/hadd.cxx

```diff
@@ -427,10 +427,10 @@ int main( int argc, char **argv )
          else
             newcomp = ROOT::RCompressionSetting::EDefaults::kUseCompiledDefault;
          delete firstInput;
-         fileMerger.SetMergeOptions(TString("first_source_compression"));
+         fileMerger.SetMergeOptions(TString("rnt:FirstSrcCompression"));
```

Comment:
I'm fine with the dot instead of colon, and I guess `rntuple` can be a bit more informative, but I like the capitalized first letter better because it makes for a cleaner mapping with the root naming style: when mapping options directly to fields of a struct you simply have to drop the `f`, and when you map to enum names you simply drop the `k` (e.g. `MergingMode=Union`).
That said, I have no strong opinion except that we should be coherent across all our API.